releng/org.eclipse.n4js.targetplatform/N4JS.setup

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it seems you completely changed the indentation of this file (auto format?) Was that done on purpose?